### PR TITLE
Add F Sharp icon

### LINKED ISSUE #5385: Request for F# icons
**Brand Name:** FSharp (F#)
 
**Website:** https://fsharp.org/

**Alexa rank:** 572629

**Official resources for icon and color:** #3B9ABF , #32C8D9

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
fsharp

**Issue:** closes #5385 

**Similarweb rank:** 1,442,604. See https://www.similarweb.com/website/fsharp.org/. The eligibility has already been debated out and was decided to be included at https://github.com/simple-icons/simple-icons/issues/5385#issuecomment-819530170

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://foundation.fsharp.org/logo
Color: `#378bba` Taken from logo.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3821,6 +3821,12 @@
             "hex": "000000",
             "source": "https://www.eyeem.com"
         },
+        {
+            "title": "F Sharp",
+            "hex": "378BBA",
+            "source": "https://foundation.fsharp.org/logo",
+            "guidelines": "https://foundation.fsharp.org/logo"
```
Comment (adamrusted) [inline _data/simple-icons.json:3828]:
```suggestion
            "title": "F#",
            "slug": "fsharp",
            "hex": "378BBA",
            "source": "https://foundation.fsharp.org/logo",
            "guidelines": "https://foundation.fsharp.org/logo",
            "aliases": {
                "aka": [
                    "F Sharp"
                ]
            }
```

@LitoMore - how do you feel about the above? It's widely referred to as `F#` and `The F# Foundation` - so ideally we should display the title as such. Have added in a slug an alias just to cover our backs.

File: icons/fsharp.svg
Comment (adamrusted) [inline icons/fsharp.svg:1]: Icon itself is perfect! Let's have a discussion around the formatting of the title, and we can get this one merged in!

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json:3828]: I had the same doubt!

Comment (adamrusted) [reply _data/simple-icons.json:3828]: Might be worth trying this change locally, and see if the linter falls out with you. What's the worst that could happen? 😅😐

Comment (LitoMore) [reply _data/simple-icons.json:3828]: `F#` looks good to me.

BTW, we have a logic for mapping special characters.

https://github.com/simple-icons/simple-icons/blob/da7cb03a790d86d3cef733095df9e60fb0de79ff/sdk.mjs#L15-L27

But I don't think we should update `TITLE_TO_SLUG_REPLACEMENTS`.

For `F# (fshrap)` or `C# (sharp)`, it's fine. But in the future universe, we may have an icon named `Simple Icons #1` or others. Its name should not be `simpleiconssharp1`.

So the extra `slug` field for this icon is reasonable.

**Review by LitoMore — APPROVED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (adamrusted) [reply _data/simple-icons.json:3828]: @PratyushJayachandran - let's GO!
If you can make the change suggested above, we'll merge it in and see if anything breaks 🤞

Comment (PratyushJayachandran) [reply _data/simple-icons.json]: Done

**Comment by PratyushJayachandran** [comment]
@adamrusted 
It is not passing checks... what to do?
```
> simple-icons@9.7.0 svglint
> svglint icons/*.svg --ci

------------------------------------- Files -------------------------------------
x icons\fsharp.svg
  x icon-title No icon with title "F Sharp" found in simple-icons.json

------------------------------------ Summary ------------------------------------
```

File: icons/fsharp.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>F Sharp</title><path d="M0 12 11.39.61v5.695L5.695 12l5.695 5.695v5.695L0 12zm7.322 0 4.068-4.068v8.136L7.322 12zM24 12 12.203.61v5.695L17.898 12l-5.695 5.695v5.695L24 12z"/></svg>
```
Comment (adamrusted) [inline icons/fsharp.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>F#</title><path d="M0 12 11.39.61v5.695L5.695 12l5.695 5.695v5.695L0 12zm7.322 0 4.068-4.068v8.136L7.322 12zM24 12 12.203.61v5.695L17.898 12l-5.695 5.695v5.695L24 12z"/></svg>
```

There's your fix @PratyushJayachandran 🙏

Comment (PratyushJayachandran) [reply icons/fsharp.svg]: I fixed it 😶‍🌫️

**Review by adamrusted — APPROVED** [review]
Amazing, thanks for persisting with this one @PratyushJayachandran 🎉🙏

**Pull request merged**